cookbooks/php/README.md

```diff
@@ -0,0 +1,11 @@
+PHP
+====
+
+This PHP recipe is used to install, configure, and eselect the specified version of PHP as well as default php.ini files. 
+
+
+## Changing the installed version
+When applicable the most strait forward way to change the PHP major version is to select the appropriate version via the dashboard.  This recipe will default to that selection first.  If you would like to change the minor version of PHP installed with this recipe this can be accomplished by updating the attributes/php.rb file.  A change made here will update both the cli and fpm php versions.
```

Comment:
Typo `strait`

Reply:
I made changes to the readme and changed that section.